dts/bindings/usb/st,stm32f7-otghs.yaml

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2025, STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32F7 OTGHS controller
+
+compatible: "st,stm32f7-otghs"
+
+include: st,stm32-otghs.yaml
+
+properties:
+  physel:
+    type: string
+    required: true
+    enum:
+      - "USB_OTG_HS_ULPI_PHY"
+      - "USB_OTG_FS_EMBEDDED_PHY"
+      - "USB_OTG_HS_EMBEDDED_PHY"
+    description: |
+      OTG_HS supports the following PHY interfaces:
+        - "USB_OTG_HS_ULPI_PHY": an UTMI+ Low Pin Interface for external high-speed PHY.
+        - "USB_OTG_FS_EMBEDDED_PHY": full speed serial transceiver select.
+        - "USB_OTG_HS_EMBEDDED_PHY": an on-chip full-speed PHY.
```

Can we harmonize the wording
```suggestion
        - "USB_OTG_HS_ULPI_PHY": UTMI+ Low Pin Interface for external high-speed PHY.
        - "USB_OTG_FS_EMBEDDED_PHY": on chip full speed PHY.
        - "USB_OTG_HS_EMBEDDED_PHY": on-chip high-speed PHY.
```
Also, on F7, I'm not seeing this `USB_OTG_HS_EMBEDDED_PHY` option (F76x/F77x). Can you cross check?

---

Done.